techmatters/flex-plugins
# CHI-970: Deduplicate flex deploy logic, improve deploy messages, bump Twilio CLI versions

Review after #1018 

Primary reviewer: @GPaoloni 

## Description

* Route all deploys via the Custom Deploy workflow, so individual deploys are consistent with mass deploys
* Add the branch / tag to deployment messaging
* Update Twilio CLI versions in the deploy action
* Split

### Checklist
- N/A New tests added
- N/A Strings are localized
- N/A Tested for chat contacts
- N/A Tested for call contacts

### Related Issues
Blocks CHI-970 (workflow deploys aren't working for flex 2.0)

### Verification steps

Use the individual & mass deploy workflows, then review the output to ensure they work as expected
